docs/api-docs/getting-started/authentication.mdx

```diff
@@ -431,8 +431,8 @@ The following table describes the authentication characteristics of our REST Pro
 
 | API description | API reference | Configuration reference | Associated API account | Authentication method |
 |:----------------|:--------------|:------------------------|:-----------------------|:----------------------|
-| [Shipping Provider API](/api-docs/providers/shipping) | [Shipping Provider Reference](/api-reference/providers/shipping-provider-api) | [Configure during registration](/api-docs/providers/shipping#configuration-fields), and/or with [Update a carrier connection](/api-reference/store-management/shipping-api/shipping-carrier/updateacarrierconnection) | implemented in an app | developer-defined request body keys | 
-| [Tax Provider API](/api-docs/providers/tax) | [Tax Provider Reference](/api-reference/providers/tax-provider-api) | [Update a tax provider connection](/api-reference/store-management/tax/tax-provider-connection/provider-connection-put) | implemented in an app | `Authorization: Basic {{base64encode(username:password)}}` |
+| [Shipping Provider API](/api-docs/providers/shipping) | [Shipping Provider Reference](/api-docs/provider-apis/shipping-provider-api) | [Configure during registration](/api-docs/providers/shipping#configuration-fields), and/or with [Update a carrier connection](/docs/rest-management/shipping-v2/shipping-carrier#update-a-carrier-connection) | implemented in an app | developer-defined request body keys | 
```

Comment:
come back there's a thing there that's changed to api docs and maybe should be reference, check all the provider links